docs/apm/troubleshooting.asciidoc

```diff
@@ -164,39 +141,35 @@ You can also use the Agent's public API to manually set a name for the transacti
 === Fields are not searchable
 
 In Elasticsearch, index templates are used to define settings and mappings that determine how fields should be analyzed.
-The recommended index template file for APM Server is installed by the APM Server packages.
-This template, by default, enables and disables indexing on certain fields.
+The recommended index templates for APM are installed by {fleet} when the Elastic APM integration is installed.
+These templates, by default, enable and disable indexing on certain fields.
 
 As an example, some agents store cookie values in `http.request.cookies`.
 Since `http.request` has disabled dynamic indexing, and `http.request.cookies` is not declared in a custom mapping,
 the values in `http.request.cookies` are not indexed and thus not searchable.
 
-*Ensure an index pattern exists*
-As a first step, you should ensure the correct index pattern exists.
-Open the main menu, then click *Stack Management > Index Patterns*.
-In the pattern list, you should see an apm index pattern; The default is `apm-*`.
-If you don't, the index pattern doesn't exist. See <<no-apm-data-found>> for information on how to fix this problem.
+*Ensure an APM data view exists*
+As a first step, you should ensure the correct data view exists.
+In {kib}, go to *Stack Management* > *Data views*.
+You should the APM data view--the default is
+`traces-apm*,apm-*,logs-apm*,apm-*,metrics-apm*,apm-*`.
```

Comment:
Is this correct?

Reply:
Going by where the default data view name is defined (https://github.com/elastic/kibana/blob/main/x-pack/plugins/apm/server/routes/data_view/get_apm_data_view_title.ts#L12-L17) it looks correct when you plug in the default values for those indices (https://github.com/elastic/kibana/blob/main/x-pack/plugins/apm/server/index.ts#L51-L54). It looks like the `apm-*` being repeated in those sections is redundant (which could be improved), but it doesn't affect the data view.

Reply:
> It looks like the apm-* being repeated in those sections is redundant (which could be improved), but it doesn't affect the data view.

Cool. That's the gist of what I was getting at with this question. Appreciate it!